test/runtests.jl

```diff
@@ -7,7 +7,7 @@ using Base.Test
 
 # Conversion and evaluation test
 @test m"expand((1+x)^2)" == m"1 + 2*x + x^2"
-@test mcall(:(exp(im*π))) == -1
```

Comment:
The native `_subst` command of maxima requires unicode, but using Julia's internal text `replace` function does not require your maxima install to be unicode.